spec/requests/posts_controller_spec.rb

```diff
@@ -313,8 +313,28 @@
         sign_in(user)
       end
 
-      it 'does not allow to update when edit time limit expired' do
+      it 'does not allow TL0 or TL1 to update when edit time limit expired' do
         SiteSetting.post_edit_time_limit = 5
+        SiteSetting.tl2_post_edit_time_limit = 30
+
+        post = Fabricate(:post, created_at: 10.minutes.ago, user: user)
+
+        user.trust_level = 1
```

Comment:
Maybe `update_columns` should do the trick it avoids a bunch of callbacks making the spec a tiny bit faster.

Reply:
Done!